templates/CRM/Group/Form/Search.tpl

```diff
@@ -157,15 +158,19 @@
         });
         //Reload table after draw
         $(settings.nTable).trigger('crmLoad');
-        if (parentsOnly) {
-          CRM.loadScript(CRM.config.resourceBase + 'js/jquery/jquery.crmEditable.js').done(function () {
+        CRM.loadScript(CRM.config.resourceBase + 'js/jquery/jquery.crmEditable.js').done(function () {
+          if (parentsOnly) {
             $('tbody tr.crm-group-parent', settings.nTable).each(function () {
               $(this).find('td:first')
                 .prepend('{/literal}<span class="collapsed show-children" title="{ts}show child groups{/ts}"/></span>{literal}')
                 .find('div').css({'display': 'inline'});
             });
+          }
+          $('tbody tr.crm-smart-group', settings.nTable).each(function () {
+            $(this).find('td:first')
+              .append(smartGroupText);
```

Comment:
Could this be simplified to:

```
$('tbody tr.crm-smart-group > td.crm-group-name', settings.nTable).append(smartGroupText);
```

(I did a bit of testing, seems to work)

Reply:
Ah right, yep it works for me too. Will update the PR.